conf/core/ConfigOptions.asciidoc

```diff
@@ -173,6 +187,13 @@ Invalid relation members will cause a warning to be logged.  You should enable t
 are loading data that has not been previously validated in another OSM API database.  Enabling this
 setting may cause writing to an OSM API database to occur more slowly.
 
+=== arff.writer.precision
```

Comment:
The default precision in the arff writer was causing some problems in my output files when read by Weka.